### ISSUE TITLE
Lichtsteuerung reparieren

### ISSUE THREAD

**Issue by abrock** [raumzeitlabor/rzl-tuwat#25]
Als ich gegangen bin habe ich versucht im Nebenraum das Licht abzuschalten, aber die Steuerung hat auf keinen Taster reagiert.
Ich habe sie mehrfach über die LSS neu gestartet, die Steuerung hat dann immer sofort alle Lampen eingeschaltet aber nicht auf die Taster reagiert.
Um das Licht zu schalten muss man jetzt also zum Schaltkasten in die Werkstatt.

**Comment by maikf** [comment]
Ja, Lichtsteuerung geht wieder, allerdings war ich etwas überfordert,
was die ganzen Sicherungen angeht.

Ich habe nicht geschafft die Steckdosen im Workshopraum wieder anzumachen.

**Comment by maikf** [comment]
Ja, Lichtsteuerung geht wieder, allerdings war ich etwas überfordert,
was die ganzen Sicherungen angeht.

Ich habe nicht geschafft die Steckdosen im Workshopraum wieder anzumachen.

Siehe auch #27

**Comment by lutoma** [comment]
Ich mache das mal wieder auf, weil die Lichtsteuerung ab und zu in den letzten Tagen noch etwas buggy war – die Situation war:

Licht im Fablab an, Licht in der Werkstatt und im Workshopraum aus. Wenn man dann auf den Lichtschalter für das Fablab drückte, ging dort zwar das Licht aus, aber dafür gingen in der Werkstatt und im Workshopraum die Lichter an (ohne dass die Lichtschalter dort irgendwie gedrückt wurden).

Das Problem scheint nicht immer aufzutreten, und ich habe noch nicht rausgefunden was genau es verursacht. Wenn es auftritt, scheint es aber ziemlich konsistent, ich habe es mehrmals hintereinander ausprobiert und es hatte immer das gleiche Verhalten. Einige andere Leute berichteten auch, dass das schon vorher so passiert sei.

Der einzige Weg in dieser Situation dann im ganzen Nebenraum das Licht auszuschalten ist, den Fablab-Schalter zu drücken um dort das Licht auszuschalten, und dann in Werkstatt & Workshopraum zu gehen und das dort nun angeschaltete Licht wieder aus zu machen.

**Comment by felicitus** [comment]
Die Lichtsteuerung schaltet nach einem Reset alle Lampen an. Verursacht wird das vermutlich durch induktive Kopplung auf die Lichtsteuerung. Die Firmware müsste mit einer Option compiled werden, damit der AVR meldet, was der Resetgrund war. Meine Vermutung damals war, daß das auf die RESET-Line des ISP-Headers eingestreut wird, scheint aber nicht der Fall zu sein. Auch scheint die Lichtsteuerung ab und zu hängen zu bleiben (Activity-LED bleibt hängen), ich weiß aber nicht, ob das related ist.
